include/psa/crypto.h

```diff
@@ -203,30 +206,51 @@ psa_status_t psa_crypto_init(void);
  */
 typedef struct psa_key_attributes_s psa_key_attributes_t;
 
-/** Declare a key as persistent.
+/** Declare a key as persistent and set its key identifier.
+ *
+ * If the attribute structure declares the key as volatile (which is
+ * the default content of an attribute structure), this function sets
+ * the lifetime attribute to #PSA_KEY_LIFETIME_PERSISTENT.
  *
  * This function does not access storage, it merely fills the attribute
- * structure with given values. The persistent key will be written to
+ * structure with given value. The persistent key will be written to
```

Comment:
```diff
- structure with given value.
+ structure with the given value.
```

or, better

```diff
- structure with given value.
+ structure with the given \p id.
```

Reply:
I don't think it's useful to specify what's going into the structure in this sentence. But I fixed the grammar and clarified a few things.